components/layout_2020/flow/inline.rs

```diff
@@ -2248,8 +2248,8 @@ struct ContentSizesComputation<'a> {
     current_line: ContentSizes,
     /// Size for whitepsace pending to be added to this line.
     pending_whitespace: Au,
-    /// Whether or not this IFC has seen any non-whitespace content.
-    had_non_whitespace_content_yet: bool,
+    /// Whether or not this IFC has seen any content, excluding discarded whitespace.
```

Comment:
Perhaps "collapsed" would be better here?

```suggestion
    /// Whether or not this IFC has seen any content, excluding collapsed whitespace.
```